app/Http/Controllers/PostsController.php

```diff
@@ -71,6 +71,11 @@ public function index(Request $request)
         $query = $this->newQuery(Post::class)
             ->orderBy('created_at', 'desc');
 
+        if ($request->query('include') && in_array('signup', $request->query('include'))) {
```

Comment:
This will break if someone provides a single include, e.g. `?include=signup`. Since this can be provided as a single include, a comma-separated list of includes, or `?include[]=` array format, let's make a `has_include($request, 'signup')` helper that can handle all these cases.

Reply:
@DFurnes super smart thanks flagging! Updated :)